mappings/net/minecraft/util/collection/Weighting.mapping

```diff
@@ -0,0 +1,14 @@
+CLASS net/minecraft/class_6007 net/minecraft/util/collection/Weighting
```

Comment:
Still Imo this is better named `Weight`, though my previous comments were lost since you've moved package